Payment UseCase/PaymentTest.java

```diff
@@ -3,6 +3,7 @@
 import static org.junit.jupiter.api.Assertions.*;
 //Testing PaymentTest class using JUnit test framework
 
+

```

Comment:
// add Log error messages and // Log info messages that cheeck invalid payment method with amassege